pkg/image/constants.go

```diff
@@ -6,7 +6,7 @@ package image
 
 const (
 	Minio          = "minio/minio:RELEASE.2021-09-15T04-54-25Z"
-	PostgresAlpine = "postgres:10.18-alpine"
-	PostgresDebian = "postgres:10.18"
+	PostgresAlpine = "postgres:10.17-alpine"
```

Comment:
figured this would be a good test to see if it works, but happy to do this another way if you have suggestions.

Reply:
Of course it works! ;)

Reply:
not your code, the github action 🤣